src/liblibc/lib.rs

```diff
@@ -6467,9 +6481,12 @@ pub mod funcs {
                                         dwMoveMethod: DWORD) -> BOOL;
                 pub fn SetEndOfFile(hFile: HANDLE) -> BOOL;
 
+                pub fn GetSystemTime(lpSystemTime: LPSYSTEMTIME);
                 pub fn GetSystemTimeAsFileTime(
                             lpSystemTimeAsFileTime: LPFILETIME);
 
+                pub fn GetLocalTime(lpSystemTime: LPSYSTEMTIME);
```

Comment:
https://msdn.microsoft.com/en-us/library/windows/desktop/ms724338(v=vs.85).aspx

Verifying that the (argument?) name is actually supposed to be lpSystemTime